packages/cadl-typescript/src/transform/transfromRLCOptions.ts

```diff
@@ -18,6 +19,9 @@ export function transformRLCOptions(program: Program, options: RLCOptions): RLCO
       config.packageDetails.scopeName = nameParts[0]?.replace("@", "");
     }
   }
+  if (options["sdk-folder"] && isAbsolute(options["sdk-folder"])) {
```

Comment:
I'd like to confirm that this is only allowed for absolute path?

Reply:
@dw511214992 will replace the value with absolute path before run cadl compile